.github/workflows/checks.yml

```diff
@@ -14,11 +14,12 @@ jobs:
       - uses: gradle/wrapper-validation-action@v1
       - uses: actions/setup-java@v1
         with:
-          java-version: 11
+          distribution: 'zulu'
+          java-version: 17
```

Comment:
Why 17 and not 18?

Reply:
https://developer.android.com/build/jdks#target-compat since its a library and not an application, I think its safer to go with 17